src/vs/workbench/contrib/terminal/browser/links/terminalWordLinkProvider.ts

```diff
@@ -40,16 +45,17 @@ export class TerminalWordLinkProvider extends TerminalBaseLinkProvider {
 		@ISearchService private readonly _searchService: ISearchService,
 		@IEditorService private readonly _editorService: IEditorService,
 		@IFileService private readonly _fileService: IFileService,
-		@IWorkbenchEnvironmentService private readonly _environmentService: IWorkbenchEnvironmentService
+		@IWorkbenchEnvironmentService private readonly _environmentService: IWorkbenchEnvironmentService,
 	) {
 		super();
+		this._xterm = _xtermTerminal.raw;
```

Comment:
Doesn't look like we need to pass `XtermTerminal` into this class?

Reply:
we need it to access the command tracker https://github.com/microsoft/vscode/blob/899aca5041c0a36906b098e119ca6445c9a6b0e4/src/vs/workbench/contrib/terminal/browser/links/terminalWordLinkProvider.ts#L193

Reply:
Oh I misread the code, in that case, let's just refer to .raw directly instead of keeping another prop?

Reply:
that's a lot of places we'll have to use .raw but ok will do